src/WebJobs.Script.ConsoleHost/Common/Functish.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace WebJobs.Script.ConsoleHost.Common
+{
+    public enum Functish
```

Comment:
yeah, we should come up with a better name :). I just needed something that could represent that enum. It's a LibGit2 lingo. They have what they call a committish, that can be commit id, branch, tag, etc. If you have a better name I'm all ears :) otherwise I'll think of something